xtask/src/ci.rs

```diff
@@ -289,16 +289,11 @@ impl CiTask {
         .map_err(Into::into)
     }
 
-    /// Lint ruma-common with clippy with the nightly version and wasm target.
-    ///
-    /// ruma-common is currently the only crate with wasm-specific code. If that changes, this
-    /// method should be updated.
+    /// Lint ruma with clippy with the nightly version and wasm target.
     fn clippy_wasm(&self) -> Result<()> {
         cmd!(
-            "
-            rustup run {NIGHTLY} cargo clippy --target wasm32-unknown-unknown
-                -p ruma-common --features api,js,rand
-            "
+            "CLIPPY_CONF_DIR=.wasm rustup run nightly cargo clippy --target wasm32-unknown-unknown
```

Comment:
I can probably remove `CLIPPY_CONF_DIR=.wasm` if i set it in `ci.yml` file?

Reply:
It's better to have it here than in `ci.yml`, that way we can use the same check locally.